d = {'Name': 'John', 'Surname': 'Smith'}

# print(d['Smith']) - We get a KeyError here, because there's no key Smith in the dictionary (Smith is a value of key Surname)

print(d['Surname'])
